### DISCUSSION TITLE
Maths Missing In Complete AI, ML, DS bootcamp ZTM?

### DISCUSSION THREAD

**Discussion by codingbeast121** [mrdbourke/zero-to-mastery-ml#109] [Q&A]
I bought the https://www.udemy.com/course/complete-machine-learning-and-data-science-zero-to-mastery/learn/lecture/38167394#overview course today and wanna start the course but before starting i have some questions :-
Do i have to learn all the sections sequencly like it is set?
And are Maths topics covered in it? Linear algebra, calculus and probability or i have to learn it seperately from somewhere else?
And should i cover these maths topics first before continuing the course from Section 2?
- Kindly answer so that i can start learning quick

**Comment by kRiShNa-429407** [comment]
No, mathematics is not covered in this course!

This course is more on code focused rather than theory!
 If you want to learn maths then check out Mathematics for Machine learning specialization on Coursera
